### PR TITLE
new translation Data-wrangling

### PR DESCRIPTION
**Pull request by vgayolrs**
Add new translation of Data wrangling and management in R, text and
images.

### CODE REVIEW

**Review by arojascastro — DISMISSED** [review]
Tiene buena pinta. Mi único comentario es que en el nombre del archivo, quizá, R debería ir en minúscula... pero como gustes.

**Comment by vgayolrs** [comment]
Tienes razón @arojascastro , no me había percatado de ello. ¿Afectará mucho si se cambia? Por otro lado, veo que no pasó ninguna de las tres verificaciones. Voy a revisar mientras el asunto.
@mdlincoln , Can you help me find the problem with the verification errors? I'm a little "rusty".

**Comment by mdlincoln** [comment]
@vgayolrs if you click on the "details" link next to where it says "continuous-integration/travis-ci/push — The Travis CI build failed" below, you will open up the check report and at the bottom in red you will see the errors (from the editor guidelines: continuous-integration/travis-ci/push — The Travis CI build failed )

It looks like one of the required YAML metadata fields is misisng.

**Comment by vgayolrs** [comment]
Thanks a lot @mdlincoln !!! Yes, I have click on "details", then i fixed some errors on links... and yes, I now can figure what the problem is. Try to fix in a while!

**Comment by vgayolrs** [comment]
@mdlincoln , YAMLfixed! But there are still some problems: it is the placement of the images. They are not in the corresponding folder but in another. I imagine several ways to proceed but what I think is the best is do that from my Git desktop (unless you suggest a more appropriate procedure).

**Comment by vgayolrs** [comment]
Thanks, @mdlincoln , I see that you moved the images but it seems that it did not work. I opened a new pull request at https://github.com/programminghistorian/jekyll/pull/1217. Maybe if you approve the review there it will be fixed.

**Comment by mdlincoln** [comment]
It did work @vgayolrs - but the image urls in the markdown file need to have the appropriate file endings (e.g. `.png`, `.jpg`) on them

**Comment by vgayolrs** [comment]
Ok @mdlincoln , working on it (and sorry for been so beginner...)

**Comment by vgayolrs** [comment]
Ok. Almost completed. There is a missing .csv file that I have already requested from the translator. https://github.com/programminghistorian/ph-submissions/issues/199 . @mdlincoln I appreciate your patience and sense of teamwork. Today I learned a lot. Thank you.

**Comment by mdlincoln** [comment]
That's what I'm here for!

File: es/lecciones/administracion-de-datos-en-r.md [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+
+```
+# Instalar el paquete historydata
+install.packages("historydata")
+
+# Cargar el paquete historydata
+library(historydata)
+```
+
+Este paquete contiene ejemplos de conjuntos de datos históricos - el ejemplo anterior con datos del censo del EEUU fue tomado de este paquete (y modificado por la traductora). A lo largo de este tutorial, vamos a trabajar con el conjunto de *early_colleges* (primeras_universidades) que contiene datos sobre las universidades fundadas antes de 1848.[^4] Lo primero que vamos a hacer es cargar los datos y leerlos:
```
Comment (mdlincoln) [inline es/lecciones/administracion-de-datos-en-r.md]: For text that references a function, file, or variable name like `early_colleges`, please use backticks like the other inline code segments.

**Review by mdlincoln — CHANGES_REQUESTED** [review]

File: es/lecciones/administracion-de-datos-en-r.md (hunk 1 above)
Comment (vgayolrs) [reply es/lecciones/administracion-de-datos-en-r.md]: Done

**Review by mdlincoln — DISMISSED** [review]

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
🎊

**Comment by jenniferisasi** [comment]
Thanks everyone!

**Pull request merged**